cmd/codegen/hack/generate_and_sort_crds.sh

```diff
@@ -24,11 +24,7 @@ FROM bitnami/python:3.10
 RUN install_packages jq
 RUN python -m pip install yq
 EOF
-    container_id=$(docker create --rm -i -v ${PWD}:${PWD}:ro -w ${PWD} -v ${tmpdir}:${tmpdir} ${image} yq $@ )
-    if [ -n "${COPY_FILE}" ] ; then
-      # When running on CI, docker-in-docker may be used, so the generated input file, which is outside the working directory, is not mounted and not available
-      docker cp "${COPY_FILE}" "${container_id}:${COPY_FILE}"
-    fi
+    container_id=$(docker create --rm -i -v ${PWD}:${PWD} -w ${PWD} -v ${tmpdir}:${tmpdir} ${image} yq $@ )
     docker start -ai "${container_id}"
```

Comment:
I think this can now be simplified to:

```suggestion
    docker run --rm -ai -v ${PWD}:${PWD} -w ${PWD} -v ${tmpdir}:${tmpdir} ${image} yq $@
```

Reply:
that fails with
```
invalid argument "i" for "-a, --attach" flag: valid streams are STDIN, STDOUT and STDERR
See 'docker run --help'.

```

Reply:
Oh right, it expects an argument. We can try just dropping it? Also, I noticed `tmpdir` is not really used here anymore, so what about removing that volume as well?:

```suggestion
    docker run --rm -i -v ${PWD}:${PWD} -w ${PWD} ${image} yq $@
```